test/kino/vega_lite_test.exs

```diff
@@ -93,6 +93,21 @@ defmodule Kino.VegaLiteTest do
     assert_broadcast_event(kino, "push", %{data: [], dataset: nil, window: 0})
   end
 
+  test "configure/2" do
```

Comment:
These tests may pollute the application environment, currently there are no other tests affected by it so I think we are ok. We could also move them into another file with `async: false` and proper cleanup on exit.

Reply:
Currently the config only affects the JS client, so all other tests should not be affected regardless that the config is, so we are good :)